website/common/script/content/faq.js

```diff
@@ -14,7 +14,11 @@ for (let i = 0; i <= NUMBER_OF_QUESTIONS; i++) {
   let question = {
     question: t(`faqQuestion${i}`),
     ios: t(`iosFaqAnswer${i}`),
-    web: t(`webFaqAnswer${i}`),
+    web: t(`webFaqAnswer${i}`, {
+      //  TODO: Need to pull these values from nconf
```

Comment:
@TheHollidayInn do you want me to revert this as well as the changes in faq.json?

I was thinking it was better as I have it since you still get benefit of decoupling the emails from the localization strings, the gotcha being that while I sort out how to pull from nconf here, these two emails (techAssistance and wikiTech) will be managed in two places: here and in the config.json file.

If I revert the changes here and the ones in faq.json, those two emails will still need to be managed in two places: config.json file and faq.json where they're hardcoded but with the added drawback that faq.json will still not have the emails decoupled from the localization strings.

So, what would you rather me do? Keep this and faq.json as is, or revert changes on those two files?